src/components/SubpageTabs.js

```diff
@@ -0,0 +1,12 @@
+document.addEventListener("DOMContentLoaded", function() {
```

Comment:
@theilman7 which one of these are we using? the seem to be doing the same thing?

Reply:
Removed the unused variable, `menuToggle`. Remaining two are in use. Just pushed up for review @martypowell